test/views/behaviours/CheckboxViewBehaviours.scala

```diff
@@ -22,10 +22,10 @@ import viewmodels.RadioCheckboxOption
 
 trait CheckboxViewBehaviours[A] extends ViewBehaviours {
 
-  def checkboxPage(form: Form[Set[A]],
-                   createView: Form[Set[A]] => HtmlFormat.Appendable,
+  def checkboxPage(form: Form[Seq[A]],
```

Comment:
if we change this behaviour to a seq not a set, should we not change all checkbox stuff to use seq instead of set as default?

Reply:
because if we generate a checkbox page now with scaffold it will fail as it passes a set as default to this behaviour.

Reply:
Correct, I had to change all set to seq. I'll update the scaffold stuff.

Reply:
Scaffold updated